static/scripts/chatLoggedin.js

```diff
@@ -0,0 +1,60 @@
+function loadChatClient(session = null) {
+	let roomIdentifier = '';
+	const matches = RegExp('/courses/([^/]+).*').exec(window.location.pathname);
+	if (matches && matches.length >= 2) {
+		roomIdentifier = `#course_${matches[1]}:matrix.stomt.com`;
+	}
+
+	// create chat tag
+	const riotBox = document.createElement('section');
+	riotBox.id = 'matrixchat';
+	riotBox.dataset.vectorIndexeddbWorkerScript = '/indexeddb-worker.js';
+	riotBox.dataset.vectorConfig = '/riot_config.json';
+	riotBox.dataset.vectorDefaultToggled = 'true';
+	riotBox.dataset.matrixRoomId = roomIdentifier;
+	riotBox.dataset.matrixLang = 'de';
+
+	if (session) {
+		riotBox.dataset.matrixHomeserverUrl = session.homeserverUrl;
+		riotBox.dataset.matrixUserId = session.userId;
+		riotBox.dataset.matrixAccessToken = session.accessToken;
+	}
+	document.body.appendChild(riotBox);
+
+	// load javascript
+	const riotScript = document.createElement('script');
+	riotScript.src = 'https://embed.stomt.com/bundles/bbf7f874b7f3891917fb/bundle.js';
```

Comment:
this needs to be moved to our server. why not to the static folder?

Reply:
good idea 👍

Reply:
The `bundle.js` file is a generated file which loads a lot of other generated files.

My plan was to move them from `embed.stomt.com` to `embed.messenger.schule`, but I am still waiting for the admins to do so.

An alternative would be to store the generated files inside the `schulcloud-client` projects `static/vendor` folder, which would be possible as well. I think we have to copy them into the nuxt client too with this solution.

What do you prefer? @janrenz @adrianjost

Reply:
I discussed the problem with @tofixx. We will host the files on an own "CDN" `embed.messenger.schule` and make it and the used version of the messenger configurable.